ConnectorRevit/ConnectorRevit/UI/ConnectorBindingsRevit.Receive.cs

```diff
@@ -235,6 +238,10 @@ void ConvertNestedElements(Base @base, ApplicationObject appObj, bool receiveDir
             converter.Report.Log(nestedAppObj);
           }
 
+          //check status, skip if necessary
+          if (nestedAppObj.Status == ApplicationObject.State.Skipped)
```

Comment:
Wouldn't the logic here be "any report object state that is not-unknown"? We wouldn't want this to run if we "Created" or "Updated" children (granted I don't think that happens but still, would be nice to prevent more duplication :)